src/OnboardComputers/OnboardComputersManager.cpp

```diff
@@ -55,6 +58,34 @@ void OnboardComputersManager::_mavlinkMessageReceived(const mavlink_message_t& m
     }
 }
 
+void OnboardComputersManager::_checkTimeouts()
+{
+    if (_onboardComputers.isEmpty())
+        return;
+
+    auto iter=_onboardComputers.begin();
+    while (iter!=_onboardComputers.end())
+    {
+        auto &compIter=iter.value();
+        // Check all computers for timeout. If has some, emitting timeout on his ID, and removing it form list.
+        if (compIter.lastHeartbeat.elapsed()>_timeoutCheckInterval*2){
+            uint8_t compID=iter.key();
+            iter=_onboardComputers.erase(iter);
+            qCDebug(OnboardComputersManagerLog)<<"Deleting onboard computer: "<<compID <<", due to timeout.";
```

Comment:
And here. Make it 
`            qCDebug(OnboardComputersManagerLog) << "Deleting onboard computer: " << compID << ", due to timeout.";`